cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/ckan/CKANBackendImpl.java

```diff
@@ -440,10 +440,11 @@ private boolean existsView(String resId) throws Exception {
         } // try catch
```

Comment:
This was already added in PR https://github.com/telefonicaid/fiware-cygnus/pull/1351